frontend/graceTunes_Vite/.eslintrc.cjs

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
this is super minor but how come the folder name is both camel case and snake case? Would prefer you to just stick to one.

Reply:
Sorry that was an oversight on my part. I'll be sticking with camel case.